tests/Doctrine/Tests/TestUtil.php

```diff
@@ -201,4 +208,61 @@ private static function mapConnectionParameters(array $configuration, string $pr
 
         return $parameters;
     }
+
+    public static function dropAllForOracle($connection) {
```

Comment:
All platform-specific code should be implemented in the DBAL, not in the ORM test suite.

Reply:
This code was added only by for Express Edition, since version dbal 3.4 it can most likely be removed already.